lib/ssl/test/ssl_crl_SUITE.erl

```diff
@@ -251,6 +257,17 @@ crl_verify_valid(Config) when is_list(Config) ->
 
     crl_verify_valid(Hostname, ServerNode, ServerOpts, ClientNode, ClientOpts),
 
+    ssl_crl_cache:insert("http://foobar/erlangCA/crl.pem", {file, filename:join([PrivDir, "otpCA", "crl.pem"])}),
+
+    R1 = ssl_crl_cache:lookup(#'DistributionPoint'{distributionPoint =
+                                                       {fullName, [{uniformResourceIdentifier, "http://foobar/erlangCA/crl.pem"}]}},
+                              undefined, {{CRLCache, internal_dummy}, internal_dummy}),
+    R2 = ssl_crl_cache:lookup(#'DistributionPoint'{distributionPoint =
+                                                       {fullName, [{uniformResourceIdentifier, "http://localhost/erlangCA/crl.pem"}]}},
+                              undefined, {{CRLCache, internal_dummy}, internal_dummy}),
+    %% Check that same path in URI does not evaluate to same result
+    true = R1 =/= R2,
```

Comment:
is this check enough? one lookup could be an error tuple and testcase would pass.
should it be verified that both results are correct? not only differ.

Reply:
Yes this enough. The problem solved is that if you added two DPs with same URI path you would only be able to retrieve one of them. The test fails before my fix and works after. This is a whitebox test.

Reply:
Also the result can not be an error tuple, it can only be not_available or some actual content.